tfhe/src/c_api/high_level_api/integers.rs

```diff
@@ -659,3 +659,112 @@ pub unsafe extern "C" fn compact_fhe_int256_list_try_encrypt_with_compact_public
 }
 
 define_all_cast_into_for_integer_type!(FheBool);
+
+macro_rules! impl_oprf_for_uint {
+    (
+        name: $name:ident
+    ) => {
+
+        ::paste::paste! {
+            #[no_mangle]
+            pub unsafe extern "C" fn [<par_generate_oblivious_pseudo_random_ $name:snake>](
```

Comment:
do we make the difference of parallel stuff at the HL level ?

Reply:
Not very useful I think, removing the `par`